doc/01_Installation.md

```diff
@@ -1,49 +1,11 @@
 # Installation
 ## Bundle Installation
-### For Pimcore >= 10.5 & \<= 10.6
-To install Pimcore Perspective Editor for Pimcore 10.5 or higher, follow the three steps below:
-
-1. Install the required dependencies:
-```bash
-composer require pimcore/perspective-editor
-```
-
-2. Make sure the bundle is enabled in the `config/bundles.php` file. The following lines should be added:
-
-```php
-use Pimcore\Bundle\PerspectiveEditorBundle\PimcorePerspectiveEditorBundle;
-// ...
-
-return [
-    // ...
-    PimcorePerspectiveEditorBundle::class => ['all' => true],
-    // ...
-];
-```
-
-3. Install the bundle:
-
-```bash
-bin/console pimcore:bundle:install PimcorePerspectiveEditorBundle
-```
-
-### For Pimcore 11
-
 You need to follow the steps mentioned above and additionally run the following command:
 
 ```bash
 composer require pimcore/admin-ui-classic-bundle
 ```
```

Comment:
this part is not necessary anymore as it is managed by composer

Reply:
Good catch I also deleted too much